tests/CMakeLists.txt

```diff
@@ -47,6 +47,9 @@ else()
     )
 endif()
 
+# Needed for VK_KHR_PORTABILITY_SUBSET_EXTENSION_NAME
+target_compile_definitions(vk_extension_layer_tests PRIVATE VK_ENABLE_BETA_EXTENSIONS)
```

Comment:
Instead of applying `VK_ENABLE_BETA_EXTENSIONS` to the whole project just apply it to the testing executable.